queries/indexing/celex-text-search.rq

```diff
@@ -0,0 +1,21 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX dc: <http://purl.org/dc/elements/1.1/>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+
+select
+?celexURI
+(group_concat(?editorialNote; separator=", ") as ?editorialNotes)
+(group_concat(?broaderConcept; separator=", ") as ?broaderConcepts)
+{
+  ?celexURI a skos:Concept .
+  optional{
+    ?celexURI skos:editorialNote ?editorialNote
+  }
+  optional {
+    ?celexURI skos:broader ?broaderConcept
```

Comment:
I would exclude the broader from the text search. 
And, instead, would add another set of editorial properties, if any available.